### DISCUSSION TITLE
Maven multimodules project and projectName

### DISCUSSION THREAD

**Discussion by dan1els** [mfussenegger/nvim-jdtls#681] [Q&A]
I guess my question is not quite original, but I'm experiencing issue with debugging multimodule maven project.
Specifically, remote session.
Everything works well, except evaluation.

The exact issue is: 
`Cannot evaluate, please specify projectName in launch.json`
Seems eclipse or whatever else failed to pass correct project name when you start remote debug session.
I can debug single module project, both maven and gradle

I guess I could pass the project name by myself, but I'm failed to find API to get it in runtime

**Comment by mfussenegger** [comment]
I'm not aware of any way to resolve the projectName automatically for remote sessions.
You'll have to configure the projectName in the configuration manually.

**Reply by dan1els** [reply]
I'm not sure how it works but for single gradle project it somehow populates automatically.
Anyway, how do you suggest to do that? Just create many configurations for each module in root .nvim.lua?
